### DISCUSSION TITLE
Welcome to chatgpt-intellij-plugin Discussions!

### DISCUSSION THREAD

**Discussion by didalgolab** [didalgolab/ai-intellij-plugin#12] [Announcements]
## 👋 Welcome to our digital neighborhood!
  This isn't a tea party. This is where the real talk happens. No fluff, no filler. Just straight-up shop talk with the ones who know the code.
  * Got a burning question that's eating you up inside? Spill it.
  * Got an idea so bright it makes the sun look like a cheap lightbulb? We wanna hear it.
  * Keep it welcoming, but don't sugarcoat it. We're all adults here. 🥃

To get started, introduce yourself, or dive right into the deep end. We're not here to judge; we're here to get things done.

**Comment by izum286** [comment]
Hello there! Firstly, I wanted to say that you're amazing! Your plugin is the only one I use in WS. My name is Ilya, and I live in Israel where I work as an R&D Senior Developer. I was wondering if it would be possible to add a feature that allows all conversations to be locally saved and organized in specific folders. Thank you!

**Reply by didalgolab** [reply]
Hi @izum286 , thanks for using the plugin, and finding it useful. Yes, conversation history is something I'm starting to work on.
